Extensions/PlatformBehavior/Extension.cpp

```diff
@@ -488,6 +521,16 @@ void DeclarePlatformBehaviorExtension(gd::PlatformExtension& extension) {
         .SetFunctionName("GetMaxFallingSpeed")
         .SetIncludeFile("PlatformBehavior/PlatformerObjectRuntimeBehavior.h");
 
+    aut.AddExpression("LadderClimbingSpeed",
+                      _("Ladder Climbing speed"),
+                      _("Get the ladder climbing speed"),
```

Comment:
A bit skeptical about this, please verify.

Reply:
Let's be consistent with the other expressions: "Ladder climbing speed"
Also careful with your casing, let's fix the name just below too, to make it "Ladder climbing speed" (no extra capitals here).